core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -814,4 +814,22 @@ class EMConfig {
     @Experimental
     @Cfg("Only used when running experiments for black-box mode, where an EvoMaster Driver would be present, and can reset state after each experiment")
     var bbExperiments = false
+
+    @Experimental
+    @Cfg("Specify whether to export covered targets info")
+    var exportCoveredTarget = false
+
+    @Experimental
+    @Cfg("Specify a file which saves covered targets info regarding generated test suite")
+    var coveredTargetFile = "coveredTargets.csv"
```

Comment:
we should have an option to be able to create a simple `txt` file, with only the unique ids of the covered targets

Reply:
fixed. this can be achieved by name option.